teps/0089-nonfalsifiable-provenance-support.md

```diff
@@ -169,22 +267,20 @@ We'll also need this feature for Tekton to achieve [SLSA 3](https://slsa.dev/lev
 * Protect against a malicious cluster admin (this will be a goal for SLSA 4)
 
 
-
 ## Proposal
 As mentioned above, the basic design looks like this (this is meant to be high level and still needs to be fleshed out a bit):
 1. Tekton Pipelines receives a TaskRun config, and generates the Pod for it with SPIRE mounted in
 1. The Pod executes, and the entrypointer requests an SVID and signature over the Results
-1. Tekton Pipelines verifies the Results
+1. Tekton Pipelines verifies the Results against the SPIRE SVID and Trust Bundle and sets the `SignedResultsVerified` condition to `True`.
 1. Meanwhile, Tekton Pipelines has been verifying that the TaskRun yaml/status hasn't been modified during execution
 1. Tekton Pipelines requests a signature and SVID over the completed TaskRun yaml from SPIRE
-1. Tekton Pipelines stores the SVID and signature as annotations on the TaskRun
+1. Tekton Pipelines stores the SVID and signature as annotations on the TaskRun status
```

Comment:
Are we proposing storing certificates and SVIDs as a result or annotations? 

https://github.com/tektoncd/community/pull/643/files?diff=split&w=1#diff-968728eb62c00df8fdd664b07424d95cfc11771904765566815a89b0c07b2296R182 says they are stored as results. 🤔

Reply:
We are storing them as annotations of the result ((https://github.com/tektoncd/pipeline/blob/34618964300620dca44d10a595e4af84e9903a55/pkg/apis/pipeline/v1beta1/taskrun_types.go#L111)). You are right as well they are put in the results of the pod - but just as a means to get to the TaskRun. 

That's just the medium of moving the information from the TaskRun Pod to the Pod Status which then moves to the TaskRun Status annotations. 

It is a bit convoluted, but that was the easiest way we could come up without complicating things too much.

Reply:
Thank you @lumjjb its reasonable to implement it this way for now and we can move to better alternative in future.